sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/DocumentIntelligenceAdministrationClient.cs

```diff
@@ -1,18 +1,18 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System.Threading.Tasks;
-using System.Threading;
 using System;
+using System.Threading;
+using System.Threading.Tasks;
 using Azure.Core;
 
 namespace Azure.AI.DocumentIntelligence
 {
     public partial class DocumentIntelligenceAdministrationClient
     {
         // CUSTOM CODE NOTE: we're overwriting the behavior of the BuildDocumentModel, ComposeModel,
-        // CopyModelTo, and BuildClassifier APIs to return an instance of TrainingOperation. This is
-        // a workaround since Operation.Id is not supported by our generator yet (it throws a
+        // CopyModelTo, BuildClassifier, and CopyClassifierTo APIs to return an instance of TrainingOperation.
+        // This is a workaround since Operation.Id is not supported by our generator yet (it throws a
         // NotSupportedException), but this ID is needed for the GetOperation API.
```

Comment:
Currently calling `Operation.Id` throws a `NotSupportedException`. This is a workaround to ensure we can extract the `Id` of LROs, since they're required for some Document Intelligence APIs.